src/components/UserProfile.jsx

```diff
@@ -35,6 +38,27 @@ export default function UserProfile({
   const metadataSchemas = useUserMetadataSchemas(userId);
   const { data: agsData, loading: agsLoading } =
     useGetUserUnprocessedAssetGroupSightings(userId);
+  const {
+    data: currentUserData,
+    loading: currentUserDataLoading,
+    error: currentUserDataError,
+  } = useGetMe();
+  const isUserManager = get(
+    currentUserData,
+    'is_user_manager',
+    false,
+  );
+  const {
+    data: nonSelfUserData,
+    loading: nonSelfUserDataLoading,
+    error: nonSelfUserDataError,
+  } = useGetUser(userId);
```

Comment:
The `userId` is not guaranteed to be nonSelf. When you view your own page, the userId is your own.

Reply:
Ended up removing this entirely.